superset/key_value/commands/update.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import logging
+from abc import ABC, abstractmethod
+from typing import Any, Dict, Optional
+
+from flask_appbuilder.models.sqla import Model
+from flask_appbuilder.security.sqla.models import User
+from sqlalchemy.exc import SQLAlchemyError
+
+from superset.commands.base import BaseCommand
+from superset.key_value.commands.exceptions import KeyValueUpdateFailedError
+
+logger = logging.getLogger(__name__)
+
+
+class UpdateKeyValueCommand(BaseCommand, ABC):
+    def __init__(
+        self, user: User, resource_id: int, key: str, data: Dict[str, Any],
+    ):
+        self._actor = user
+        self._resource_id = resource_id
+        self._key = key
+        self._properties = data.copy()
+
+    def run(self) -> Model:
+        try:
+            value = self._properties.get("value")
```

Comment:
Same comment as the `Create` case. Let's just use the `data` passed in via the ctor directly

Reply:
Done